debian/control

```diff
@@ -39,15 +52,21 @@ Description: snappy development go packages.
 Package: snapd
 Architecture: any
 Depends: adduser,
+         apparmor (>= 2.10.95-0ubuntu2.2),
          gnupg1 | gnupg,
-         snap-confine (>= 1.0.43),
          squashfs-tools,
          systemd,
          ${misc:Depends},
          ${shlibs:Depends}
-Replaces: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9)
-Breaks: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9)
-Conflicts: snap (<< 2013-11-29-1ubuntu1)
+Replaces: snap-confine,
```

Comment:
Can we please just make sure the versions are locked down?  snapd X requires snap-confine X, exactly. That removes potential problems we've observed before for having them out-of-sync.

Anything else we need to be careful about if we do that?